keps/sig-node/3695-pod-resources-for-dra/README.md

```diff
@@ -0,0 +1,420 @@
+been injected into a container are not tied to the "availability" of the DRA
+driver that injected them -- i.e. once a DRA driver allocates a set of CDI
+devices to a container, that container will have full access to those devices
+for its entire lifetime (even if the DRA driver that injected them temporarily
+goes offline). In this way, the in-memory cache maintained by the `DRAManager`
+will always have the most up-to-date information for all running containers (so
+long as checkpointing is added as described to repopulate it across kubelet
+restarts).
+
+##### Unit tests
+
+- `k8s.io/kubernetes/pkg/kubelet/apis/podresources`: `01-24-2023` - `61.5%`
+
+##### Integration tests
+
+ These cases will be added in the existing integration tests:
+  - Feature gate enable/disable tests.
+  - Get API work with DRA and device plugin.
+  - List API work with DRA and Device plugin.
+
+##### e2e tests
+
+These cases will be added in the existing e2e tests:
+  - Feature gate enable/disable tests.
+  - Get API work with DRA and device plugin.
+  - List API work with DRA and Device plugin.
+
+### Graduation Criteria
+
+#### Alpha
+
+- [ ] Feature implemented behind a feature flag.
+- [ ] e2e tests completed and enabled.
+
+#### Beta
+
+- [ ] Gather feedback from consumers of the DRA feature and k8snetworkplumbingwg working group
+- [ ] No major bugs reported in the previous cycle.
+
+#### GA
+
+- [ ] Allowing time for feedback (1 year).
+- [ ] Risks have been addressed.
+
+### Upgrade / Downgrade Strategy
+
+With gRPC the version is part of the service name.
+Old versions and new versions should always be served and listened by the kubelet.
+
+To a cluster admin upgrading to the newest API version, means upgrading Kubernetes to a newer version as well as upgrading the monitoring component.
+
+To a vendor changes in the API should always be backwards compatible.
+
+### Version Skew Strategy
+
+Kubelet will always be backwards compatible, so going forward existing plugins are not expected to break.
+
+
+## Production Readiness Review Questionnaire
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name for retrieving resources allocated by DRA: `DynamicResourceAllocation` and `PodResourcesDynamicResources`.
+  - Feature gate name for Get method: `PodResourcesGet`. In case `DynamicResourceAllocation` or the `PodResourcesDynamicResources`
+    are disabled and `PodResourcesGet` is enabled, the Get method will retrieve resources allocated by device plugins, memory and cpus (but omit those allocated by DRA resource drivers).
+    In case `PodResourcesGet`, `DynamicResourceAllocation` and `PodResourcesDynamicResources` are all enabled, the Get method will also retrieve the resources allocated via DRA.
+  - Components depending on the feature gate: kubelet, 3rd party consumers.
```

Comment:
Would you mind state clear on 
1. which FG is reused from existing and which is proposed to be added. 
2. For each feature gate, which part it tries to gate?
3. Component depending on each feature gate separately?

Reply:
I update the kep to be clearer on the feature gates.